src/ciadmin/generate/worker_pools.py

```diff
@@ -201,109 +201,77 @@ def get_azure_provider_config(
             "wstServerURL", azure_config["wst_server_url"]
         )
     tags = config.get("tags", {})
-    # Temporary conditional language while we move pools over to the newer
-    # method that is in the else clause
-    # Refrence https://mozilla-hub.atlassian.net/browse/RELOPS-212
-    if config.get("old-style-worker-config"):
```

Comment:
Thanks for deleting this! Though it would be nice to remove this in a separate PR (or at least a separate commit) to make the diff easier to read.

Reply:
Good point. I am going to separate this into different PRs.